LICENSE

```diff
@@ -1,4 +1,5 @@
-(c) 2014 Wedy Chainy <askwedi@wedipedia.org>
+Copyright (C) BigCommerce, 2014
```

Comment:
Nitpick, but I believe the official name is `Bigcommerce` these days. You should probably check with someone more knowledgeable than me though.